src/Carbon/Lang/it.php

```diff
@@ -43,6 +43,9 @@
     'second' => ':count secondo|:count secondi',
     'a_second' => 'alcuni secondi|:count secondi',
     's' => ':count sec.',
+    'millisecond' => '{1}:count millisecondo|{0}:count millisecondi|]1,Inf[:count millisecondi',
+    'a_millisecond' => '{1}pochi millisecondi|{0}:count millisecondi|]1,Inf[:count millisecondi',
```

Comment:
You can rely on the `|` for the plural without specifying `{0}` `{1}`, the correct pluralization rules are provided by symfony/translation.